.github/workflows/tests.yml

```diff
@@ -61,7 +61,7 @@ jobs:
           verbose: true
 
   tests:
-    name: ${{ matrix.continuum }} continuum ${{ matrix.rpacket_tracking }} rpacket_tracking ${{ matrix.os }} ${{ inputs.pip_tests && 'pip tests enabled' || '' }}
+    name: ${{ matrix.continuum }} continuum ${{ matrix.rpacket_tracking }} rpacket_tracking ${{ matrix.os }} ${{ inputs.pip_git && 'pip tests enabled' || '' }}
```

Comment:
What does this change do?

Reply:
this was an error in the yml file which I'm afraid I didn't see. The job name will now say `pip tests enabled` if those tests are triggered by the pre release workflow. If not it will be as it is now.

Reply:
this was an error in the yml file which I'm afraid I didn't see. The job name will now say `pip tests enabled` if those tests are triggered by the pre release workflow. If not it will be as it is now.